root/static/scripts/account/components/PreferencesForm.js

```diff
@@ -189,7 +189,7 @@ class PreferencesForm extends React.Component<Props, State> {
         <fieldset>
           <legend>{l('Privacy')}</legend>
           <p>
-            {l('Allow other users to see:')}
+            {addColonText(l('Allow other users to see'))}
```

Comment:
Should we always be using `addColonText` then even if there's not (and we don't expect there will be) a non-colon equivalent string? I'm fine with that, just wasn't sure that was a thing :)

Reply:
Yes, even though it is probably not worth breaking all the existing translations for long-standing source strings at the moment.